pkg/apis/pipeline/v1alpha1/pod.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2018 The Tekton Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+)
+
+type PodTemplate struct {
```

Comment:
Relate to https://github.com/tektoncd/catalog/pull/50#discussion_r300458884, I think I should add `Volumes` here also (to be able to attach volume at runtime)